lms/static/js/learner_dashboard/views/explore_new_programs_view.js

```diff
@@ -0,0 +1,44 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/explore_new_programs.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             exploreTpl
+         ) {
+            return Backbone.View.extend({
+                el: '.program-advertise',
+
+                tpl: _.template(exploreTpl),
+
+                initialize: function(data) {
+                    this.context = data.context;
+                    this.$parentEl = $(this.parentEl);
+
+                    if (this.context.xseriesUrl){
+                        // Only render if there is an XSeries link
+                        this.render();
```

Comment:
Both are good points I guess. I am fine this PR doing it as it is. However, going forward, should we maintain a consistent method to always call `render()` in the init function unless there is an explicit need not to do so?